### DISCUSSION TITLE
Issue getting data to wrap

### DISCUSSION THREAD

**Discussion by DWLMARKPICKELL** [kinverarity1/lasio#540] [Q&A]
Using LASIO to convert CSV data to LAS format and everything is working great but I just can't seem to get the data to WRAP.  In my code I'm setting the WRAP to YES using

las.version.WRAP = 'YES'
This is followed by setting all the headers info, pulling data and then las.write(file,version=vers)

This results in my las headers showing: (which las.version.WRAP confirms)
~Version ---------------------------------------------------
VERS.   2.0 : CWLS log ASCII Standard -VERSION 2.0
WRAP.   YES : One line per depth step
DLM . SPACE : Column Data Section Delimiter
~Well ------------------------------------------------------

The data though will not wrap and when running through the lascertify application it fails stating that the header indicates WRAPPED but the data is not in fact wrapping.  Observation of file confirms this.   Happens in both Ver 1.2 and 2.0

**Comment by kinverarity1** [comment]
Sorry I never got back to your question - this should work as you describe. I'll have a look into it when I get a chance.
